common/containers/Tabs/SendTransaction/index.jsx

```diff
@@ -238,8 +267,63 @@ export class SendTransaction extends React.Component {
     return { to, data, value, unit, gasLimit, readOnly };
   }
 
+  isValid() {
+    const { to, value } = this.state;
+    return (
+      isValidETHAddress(to) &&
+      value &&
+      Number(value) > 0 &&
+      !isNaN(Number(value)) &&
+      isFinite(Number(value))
+    );
+  }
+
+  // FIXME MOVE ME
+  getTransactionFromState() {
+    // FIXME add gas price
+    if (this.state.unit === 'ether') {
+      return {
+        to: this.state.to,
+        from: this.props.wallet.getAddress(),
+        // gasPrice: `0x${new Number(50 * 1000000000).toString(16)}`,
+        value: '0x' + toHex(toWei(Big(this.state.value), 'ether'))
```

Comment:
We should ban `number` usage altogether just in case. 0x prefixes are sometimes needed and sometimes not we need to better type this with https://flow.org/en/docs/types/opaque-types/